partner_firstname/readme/CONTRIBUTORS.rst

```diff
@@ -18,3 +18,6 @@
 * Graeme Gellatly <graeme@o4sb.com>
 * Laurent Mignon <laurent.mignon@acsone.eu>
 * Bjorn Billen <bjorn.billen@dynapps.be>
+* `Aion Tech <https://aiontech.company/>`_:
```

Comment:
This syntax seems to not be correct, I'd include only the contributor name, as in the already-existing list items.

Reply:
Why is it not correct? It looks fine in the README: 
![image](https://github.com/OCA/partner-contact/assets/143788994/95356d2a-a999-4b55-ab0e-c282c666fe90)